File: SistemaVotacion2/SistemaVotacion2/Models/Candidato.cs
using System;
using System.Collections.Generic;
using SistemaVotacion2.Models;

namespace SistemaVotacion2.Models
{
    public class Candidato
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Partido { get; set; }

        public ICollection<Voto> Votos { get; set; }
    }
}

File: admElectoral/admElec/models/candidato.py
from django.db import models
from .partido import Partido
from .cargo import Cargo

class Candidato(models.Model):
    nombre = models.CharField(max_length=100)
    partido = models.ForeignKey(Partido, on_delete=models.CASCADE)
    cargo = models.ForeignKey(Cargo, on_delete=models.CASCADE)

    def __str__(self):
        return self.nombre
